ci-operator/config/openshift/verification-tests/openshift-verification-tests-main__ota-multi-stable-4.21-upgrade-from-stable-4.20.yaml

```diff
@@ -0,0 +1,130 @@
+base_images:
+  ansible:
+    name: "4.20"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.20"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.20"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.21"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.20"
+  tools:
+    name: "4.20"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.20"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+releases:
+  intermediate1:
+    prerelease:
+      architecture: multi
+      product: ocp
+      relative: 1
+      version_bounds:
+        lower: 4.21.0-0
+        stream: 4-dev-preview
+        upper: 4.22.0-0
+  latest:
+    release:
+      architecture: multi
+      channel: candidate
```

Comment:
It is necessary to use `fast` channel to align with 4.20. `candidate` is too new to be a source version for particular test cases in this job.